packages/lesswrong/lib/publicSettings.ts

```diff
@@ -85,3 +85,7 @@ export const defaultVisibilityTags = new DatabasePublicSetting<Array<FilterTag>>
 
 export const gatherTownRoomId = new DatabasePublicSetting<string | null>("gatherTownRoomId", "aPVfK3G76UukgiHx")
 export const gatherTownRoomName = new DatabasePublicSetting<string | null>("gatherTownRoomName", "lesswrong-campus")
+
+// Public elicit settings
+export const elicitSourceURL = new DatabasePublicSetting('elicitSourceURL', 'https://LessWrong.com')
```

Comment:
```suggestion
export const elicitSourceURL = new DatabasePublicSetting('elicitSourceURL', 'https://www.lesswrong.com')
```

Reply:
Sadly, the API key is registered with that URL, so we can't change it without requesting a new API key.

Reply:
Alas. This may cause problems some day (if it ever uses the URL in a way that the lesswrong.com->www.lesswrong.com redirect can't handle) but is probably fine.